injector/src/main/java/edu/ucr/cs/riple/injector/Helper.java

```diff
@@ -324,6 +329,34 @@ public static NodeList<BodyDeclaration<?>> getTypeDeclarationMembersByFlatName(
     return getMembersOfNode(cursor);
   }
 
+  /**
+   * Locates a variable declaration expression in the tree of a {@link CallableDeclaration} with the
+   * given name.
+   *
+   * @param encMethod The enclosing method which the variable is declared in.
+   * @param varName The name of the variable.
+   * @return The variable declaration expression, or null if it is not found.
+   */
+  @Nullable
+  public static VariableDeclarationExpr locateVariableDeclarationExpr(
```

Comment:
It seems we do not yet support local variables inside initializer blocks or lambdas; is that correct?  If so we should probably document it somewhere.

Reply:
Yes, documented the limitation. 8b433a6cd95a4d4709ced08167f8c5d16e7b48ca

Reply:
Do we have a test involving a local inside a lambda or initializer expression?  Just want to make sure we don't crash.

Reply:
@msridhar Sure, please find the test here 5d14a4cbbbdbf22d649bf891a5bce3ff0a0c0b32
Please let me know if the test is sufficient.

Reply:
LGTM!